### PR TITLE
Fix/paths

### PR DESCRIPTION
**Pull request by hellais**
This also includes a fix that removes some invalid metadata from the setup.py

### CODE REVIEW

File: ooni/settings.py [hunk 1]
```diff
@@ -156,6 +156,9 @@
 
 # These are the the embedded settings
 _SETTINGS_INI = os.path.join(OONIPROBE_ROOT, 'settings.ini')
+# This can be overridden by an environment variable for end to end testing
+if 'OONIPROBE_SETTINGS' in os.environ:
+    _SETTINGS_INI = os.environ['OONIPROBE_SETTINGS']
```
Comment (darkk) [inline ooni/settings.py]: `os.environ.get('OONIPROBE_SETTINGS', os.path.join(OONIPROBE_ROOT, 'settings.ini'))` is a bit more pythonic :)

File: setup.py [hunk 2]
```diff
@@ -110,7 +110,6 @@
 Intended Audience :: Information Technology
 Intended Audience :: Science/Research
 Intended Audience :: Telecommunications Industry
-License :: OSI Approved :: BSD Licens
```
Comment (darkk) [inline setup.py]: Was it enough to add `e` to `Licens` to make it valid?

File: ooni/settings.py (hunk 1 above)
Comment (hellais) [inline ooni/settings.py]: Yes that is true, but I prefer to have it be explicit and clear that this is what is going on here. Moreover we may want to override other settings via environment variables and it's good to have a consistent pattern for doing so.

File: setup.py (hunk 2 above)
Comment (hellais) [inline setup.py]: Yeah that's true. I can fix it that way too.

**Comment by darkk** [comment]
LGTM

**Pull request merged**